tests/fixtures/explore.ts

```diff
@@ -20,6 +25,17 @@ export class ExplorePage {
     this.refreshPicker = this.page.getByTestId(testIds.header.refreshPicker)
   }
 
+  async setDefaultViewportSize(){
+    await this.page.setViewportSize({ width: 1280, height: 720 });
+  }
+
+  async setLimoViewportSize(){
```

Comment:
Limo?

Reply:
haha I was being silly, I'll clean up tomorrow

Reply:
Haha no worries, it looked like a typo.

Reply:
Now I get it, with the 3000 height.

Reply:
Made it a bit more serious :)